python/cugraph/structure/graph.py

```diff
@@ -116,6 +118,81 @@ def clear(self):
         self.adjlist = None
         self.transposedadjlist = None
 
+    def add_nodes_from(self, nodes, bipartite=None, multipartite=None):
```

Comment:
Are we adding a notebook separately? 

How about documentation on bipartite graphs, when to use them instead of regular cugraph graphs, and what is supported? Something along the lines of https://github.com/rapidsai/cugraph/blob/branch-0.15/docs/source/dask-cugraph.rst?

Reply:
agree on docs and sample notebook

Reply:
We can add the docs within the code as now. As for notebooks i believe we should first add some bipartite datasets to cugraph to run on some algorithms to demonstrate its actual usage